client/src/main/java/com/paypal/selion/reports/runtime/Gatherer.java

```diff
@@ -41,7 +44,10 @@ static String saveGetLocation(WebDriver driver) {
                 location = driver.getCurrentUrl();
             }
         } catch (Exception exception) {
-            logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
+            logger.log(
```

Comment:
"How about checking the exception message in the catch block for Not yet implemented ... "

I started going down that route but different drivers will throw a different message.
Appium-IOS does : Not implemented in this context, try switching into or out of a web view
Appium-Android does: Not yet implemented. Please help us: http://appium.io/get-involved.html
IOS-driver: